src/main/java/catan/services/util/play/PlayUtil.java

```diff
@@ -48,7 +47,7 @@ public void updateAvailableUserActions(GameBean game) throws GameException {
                 preparationStageUtil.updateAvailableUserActions(game);
```

Comment:
this method also update actions for all users, so name should be the same as for main stage: updateAvailableActionsForAllUsers